usefulUtilities/sit/sitClient.js

```diff
@@ -134,36 +137,37 @@
     }
 
     var UPDATE_INTERVAL_MS = 400;
-    var isConnected = false;
+    var isActionEventConnected = false;
+    var isSittingInThisChair = false;
     function sitDownAndPinAvatar() {
         if (HMD.active) {
             deletePresit();
         }
-        if (!isConnected) {
+        if (!isActionEventConnected) {
             Controller.actionEvent.connect(onActionEvent);
-            isConnected = true;
+            isActionEventConnected = true;
         }
-        Script.setTimeout(function () {            
-            var properties = Entities.getEntityProperties(_this.entityID);
-            MyAvatar.beginSit(_this.seatCenterPosition, properties.rotation);
-            
-            _this.sitDownSettlePeriod = Date.now() + SIT_SETTLE_TIME_MS;
-            stopUpdateInterval();
-
-            _this.whileSittingUpdateIntervalID = Script.setInterval(_this.whileSittingUpdate, UPDATE_INTERVAL_MS);
-
-            deleteSittableUI();
 
-            if (!_this.connectedSignals) {
-                MyAvatar.scaleChanged.connect(_this.standUp);
-                MyAvatar.onLoadComplete.connect(_this.standUp);
-                location.hostChanged.connect(_this.standUp);
-                Script.scriptEnding.connect(_this.standUp);
-                MyAvatar.wentAway.connect(_this.standUpWrapper);
-                HMD.displayModeChanged.connect(_this.standUpWrapper);
-                _this.connectedSignals = true;
-            }
-        }, SIT_DELAY_MS);
+        var properties = Entities.getEntityProperties(_this.entityID);
```

Comment:
It looks like we only use rotation so can we limit the request to just that property?

Reply:
ACK